examples/step-37/doc/intro.dox

```diff
@@ -473,3 +484,44 @@ settings (on the command line, specify
 <tt>-DCMAKE_CXX_FLAGS="-march=native"</tt>, see the deal.II README for more
 information). Similar options exist for other compilers.
 
+
+<h3>Running multigrid on large-scale parallel computers</h3>
+
+As mentioned above, all components in the matrix-free framework can easily be
+parallelized with MPI using domain decomposition. Thanks to the good support
+of large-scale parallel mesh framework provided by p4est (see step-40 for
+details) in deal.II, and the fact that cell-based loops with matrix-free
+evaluation <i>only</i> need a decomposition of the mesh into chunks of roughly
+equal size on each processor, there is relatively little to do for extending
+the program to MPI. While other tutorial programs using MPI have relied on
+either PETSc or Trilinos, this program uses deal.II's own parallel vector
+facilities.
+
+The deal.II parallel vector class, LinearAlgebra::distributed::Vector, holds
+the processor-local part of the solution as well as information on and data
+fields for the ghost DoFs, i.e. DoFs that are owned by a remote processor but
```

Comment:
@kronbichler is this documented somewhere? If not, maybe copy-paste this into `MatrixFree` class and add a link from the glossary?